pkg/pow/solutionHandler.go

```diff
@@ -51,73 +52,62 @@ func CreateSolutionHandler(cfg *config.Config, submitter tellorCommon.Transactio
 	}
 }
 
-func (s *SolutionHandler) Submit(ctx context.Context, result *Result) bool {
+func (s *SolutionHandler) Submit(ctx context.Context, result *Result) (*types.Transaction, error) {
```

Comment:
need to return the transaction data to use the gas cost when saving the tx cost in the db.